svf/include/Util/Annotator.h

```diff
@@ -112,7 +112,7 @@ class Annotator
             values.push_back(val);
             //llvm::ArrayRef<llvm::Metadata*> ar(metavalues);
             // FIXME: delete the old MDNode
-            inst->setMetadata(str, MDNode::get(inst->getContext(), llvm::None));
+            inst->setMetadata(str, MDNode::get(inst->getContext(), nullptr));
```

Comment:
good fix!

Reply:
Could you make this fix as a separate pull request?

Reply:
Could you make another PR with this line of fix only, so that I could merge it first?

Reply:
https://github.com/SVF-tools/SVF/pull/1094